src/libraries/Microsoft.Extensions.HostFactoryResolver/src/Microsoft.Extensions.HostFactoryResolver.Sources.csproj

```diff
@@ -10,6 +11,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="*.cs" PackagePath="contentFiles/cs/netstandard1.0/%(FileName)%(Extension)" />
+    <Content Include="*.cs" PackagePath="contentFiles/cs/netstandard1.0/%(FileName)%(Extension)" BuildAction="Compile" />
```

Comment:
@Anipik I checked the generated package's nuspec and before this change there were two items added as following:
- one content item for netstandard1.0 without a compile build action
- one compile item added for netstandard2.0

I assume that the content item was unintentional? The package wouldn't have worked and participated in the user's build for netstandard1.x targets. That said, I plan to trim assets from our packages which includes netstandard1.x assets so this might go away entirely.

Reply:
> I assume that the content item was unintentional?

thats unintentinal